res/TensorFlowLiteRecipes/Quant_Add_000/test.recipe

```diff
@@ -0,0 +1,36 @@
+operand {
+  name: "ifm"
+  type: UINT8
```

Comment:
Isn't it FP32?

Reply:
u8 is what I intended. This recipe is used to test fake quantization (quantized model -> fake-quantized model (fp32)).